blacken_docs.py

```diff
@@ -32,6 +32,7 @@
     rf'(?P<before>'
     rf'^(?P<indent> *)\.\. ('
     rf'jupyter-execute::|'
+    rf'invisible-code-block::?|'
```

Comment:
this would handle _all_ code blocks which isn't correct -- it needs to specifically be python code blocks